mythril/laser/ethereum/transaction.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from mythril.laser.ethereum.state import GlobalState, Environment, CalldataType
+from mythril.laser.ethereum.cfg import Node, Edge, JumpType
+from z3 import BitVec
+
+
+class CallTransaction:
```

Comment:
I think `MessageCall` would be a  more accurate name for this class.

Reply:
I'll change it